hv-rhel6.x/hv/vmbus_drv.c

```diff
@@ -87,6 +87,50 @@ struct hv_device_info {
 	struct hv_dev_port_info outbound;
 };
 
+#if defined(RHEL_RELEASE_VERSION) && (RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(6,4))
+
+#define __is_kfree_rcu_offset(offset) ((offset) < 4096)
+
+/*
+ * Helper macro for kfree_rcu() to prevent argument-expansion eyestrain.
+ */
+#define __kfree_rcu(head, offset) \
+	do { \
```

Comment:
I suppose symbol kfree_rcu doesn't exist in old kernel hence you adding it.
Did you try to find how similar operation done without this symbol?

Reply:
I have modified git commit message. kfree_rcu was introduced by the commit c6100fd. 
The kfree_rcu is a Kernel symbol for newer Kernels and does not present in older kernel, so added the kfree_rcu symbol in LIS to handle older Kernels build issues. 
I did look into for a similar operation in older kernel as that could have needed to rewrite the vmbus_chan_release().